views/reports/custom_reports.php

```diff
@@ -19,11 +19,17 @@
 	
 	global $db;
 
-	$query = "SELECT DISTINCT(curriculumname), curriculumid FROM curricula;";
+	$query = "SELECT DISTINCT(curriculumname), curriculumid FROM curricula WHERE df <> 1;";
```

Comment:
While the df flag used to be 0 or 1 it's been updated to be `BOOLEAN` [here](https://github.com/Capping-CPCA/Database/blob/4cce0d1b5169677315c3a2417c32febcfccea3a1/TABLES.sql#L228).

Reply:
So 10.11.12.33 has the new DB and should be hosting the latest version of the database. The correct way to check should be `WHERE df IS FALSE`

Here's a snippet from the postgres documentation:
![image](https://user-images.githubusercontent.com/9338669/33337307-9506a2fa-d440-11e7-96b0-6d21133914a3.png)

Reply:
Okay cool, that makes way more sense. Updating now.